manifests/init.pp

```diff
@@ -102,18 +102,19 @@
     ssl_cert_db            => $::certs::nss_db_dir,
     ssl_cert_password_file => $::certs::qpid::nss_db_password_file,
     ssl_cert_name          => 'broker',
+    interface              => 'lo',
```

Comment:
Right, so I was missing something. Not sure why I thought this was a class parameter ```katello``` instead of ```qpid```. My bad.